### DISCUSSION TITLE
Is the next version expected within 22 years?

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/thorium#99] [General]
More than two months!? 
It has violated the setting that thorium construction is above the top of Google tree. Microsoft Edge Canary is 110.0 and Firefox is 110.0. Only thorium construction has been waiting so long?

**Comment by gz83** [comment]
> More than two months!? It has violated the setting that thorium construction is above the top of Google tree. Microsoft Edge Canary is 110.0 and Firefox is 110.0. Only thorium construction has been waiting so long?
@whxuezhe 
I have answered you before, please do not repeat this question.
